docs/src/snap/howto/cis-hardening.md

```diff
@@ -22,224 +22,11 @@ This guide assumes the following:
 - You have root or sudo access to the machine
 
 
-## Post-deployment extra hardening steps
-
-The following hardening configurations are not part of the default {{product}}
```

Comment:
I believe that this section is better suited remaining in this page. The purpose of this how to guide is to become cis compliant and get kube-bench to pass without warnings. Jumping from page to page may be confusing.

Reply:
We can use an {include} statement if you want to reduce maintenance having it in two locations